### ISSUE TITLE
Persistent Chunk Error

### ISSUE THREAD

**Issue by isaiahshiner** [jellysquid3/Sodium#1079]
### Version information

0.4.0-alpha5+build.9

### Expected Behavior

Normal chunk rendering, or at least for the chunk error to disappear on relogging.

### Actual Behavior

A 4x3 chunk area was invisible for all players running Sodium. The blocks were fully present and interactable and visible to vanilla players. It persisted after leaving the area and returning, F3+A, relogging, restarting client, restarting machine, and switching to a Vanilla instance and back. The glitch finally disappeared on its own after about an hour, but presumably, it also would have disappeared on a server restart.

I was originally running Sodium, Lithium and Phosphor. Two more players were running Sodium and Iris, and another just Sodium. We all saw the error. It persisted when I switched to just Sodium.

### Reproduction Steps

??? I will add more information if it happens again.

### Java version

Java 17.0.1 64 bit

### CPU

Intel i7-1065G7

### GPU

Intel Iris (the intel product, not the shader mod) Plus Graphics 640

### Additional information

Lithium 0.7.6
Phosphor 0.8.1
Minecraft 1.18.1
Paper Server 94

**Comment by mmvanheusden** [comment]
happens here as well, i believe it also breaks block breaking

**Comment by Remiculus** [comment]
I believe I've experienced this as well. specifically on Wynncraft. unfortunately, it put a damper on my first impressions of the sever but it wasn't their fault. it's the same chunks every time every which way I've tried. oddly enough I haven't experienced this on other servers. would you like me to attach screenshots?

**Comment by edwardg117** [comment]
Expirenced this yesterday, also affected vanilla clients. Running C2ME, Chat+, Chunk Pregenerator, flytre lib, Lithium, Starlight, Terralith and Simple Voice Chat on server-side. I excpect either a conflict, an issue with one of these or a 1.18 specific issue.
TL;DR: Vanilla client also affected by this probably means it's not a sodium issue.

**Comment by yuiidev** [comment]
Try switching off your resourcepacks if you're using any. I had this happen with just a 1x1 chunk area but several locations which seemed to be caused by Sodium not supporting that resourcepack properly.

The bug doesn't happen in vanilla with the resourcepack ([Alacrity](https://www.curseforge.com/minecraft/texture-packs/alacrity)) turned on.

**Comment by isaiahshiner** [comment]
> Vanilla client also affected by this

Yes, might not be this exact issue, as the error did not appear in vanilla.
> Try switching off your resourcepacks if you're using any

I was not using any. I think some of those affected may have had resourcepacks, I'm not sure.

In any case, I have not seen a repeat of the bug anywhere since the first time.

**Comment by marlester-dev** [comment]
This error is still present and Sodium doesn't fix it, can confirm in Sodium 0.4.4

**Comment by marlester-dev** [comment]
It's verry annoying
